src/app/item-page/simple/item-page.component.html

```diff
@@ -3,6 +3,16 @@
     <div class="item-page" @fadeInOut>
       @if (itemRD?.payload; as item) {
         <div>
+          <ng-container *ngVar="(itemRequest$ | async) as itemRequest">
+            @if (hasValue(itemRequest)) {
+              <div class="alert alert-warning wb-100 mb-2">
+                <p><span role="img" class="request-a-copy-access-icon"><i class="fa-solid fa-lock-open"></i></span>{{'bitstream-request-a-copy.access-by-token.warning' | translate}}</p>
```

Comment:
The open lock icon needs to have `alt` text added.  Currently, if you scan this page with the "axe DevTools" plugin for Chrome, it gives this error:
```
Ensure [role="img"] elements have alternative text
```

Reply:
Done for this and bitstream link, using `[attr.aria-label]` so a i18n key can be used.